### ISSUE TITLE
Faster loot

### ISSUE THREAD

**Issue by Try** [Try/OpenGothic#11]
1. If player hold action-button, the number of items to take has to be increased *10, for every 10 take action
2. Need to implement take *10, take *100 hotkeys

**Comment by Kirides** [comment]
Is looting via the "Jump" button already implemented? (Loot whole stack of items with ALT-Key per default)

**Comment by Try** [comment]
Only most simple looting with Action-Button implemented for now

**Comment by da3m0nsec** [comment]
What do you think is the best approach for implementing x10 and x100 hotkeys? I thought about adding a variable to the class for indicating the looting mode, and then using it in on onTakeStuff()
```cpp
enum class LootingMode:uint8_t {
      Normal=0,
      Stack,
      Ten,
      Hundred
      };

LootingMode     actualLootingMode = LootingMode::Normal;
```

```cpp
if(e.key==KeyEvent::K_LOOTTEN){
    actualLootingMode = Ten;
    takeTimer.start(100);
    onTakeStuff();
    }
```
But that could introduce some inconveniences like having to add differents onTakeStuff functions (onTakeTen ?) or some big if's, as, for example, we don't want them to be affected by looting speed increasing right?
Any idea for a better implementation?

**Comment by da3m0nsec** [comment]
Implemented on #41
